test-support/page-object.js

```diff
@@ -75,3 +77,8 @@ export default {
 };
 
 export { buildSelector, findElementWithAssert, findElement, getContext, fullScope } from 'ember-cli-page-object';
+
+deprecate(`Importing from "test-support" is now deprecated. Just import directly from the "ember-cli-page-object" module instead.`, false, {
```

Comment:
This is a total unimportant nitpick, but suggest replacing `just` with `please` here; I think that's more conventional language for deprecation messages.